### ISSUE TITLE
pRETARINED model you have generated is for CycleGAN or Pix2Pix?

### ISSUE THREAD

**Issue by chowkamlee81** [sauradip/night_image_semantic_segmentation#5]
Hi Sauradip, @sauradip 

I had to convert from day time to night time. The pretrained model you had trained yourself is wrt CycleGAN or pix2pix. Kindly requesting your help on this and do the needful

**Comment by sauradip** [comment]
Hi @chowkamlee81 , the pretrained model is wrt CycleGAN , since we have no pair of image for day and night for same image , hence mapping is missing which leads us to train a unsupervised Domain Adaptation technique which is CycleGAN . Pix2Pix will work if you have a day and night for the same image i.e supervised domain adaptation.

**Comment by chowkamlee81** [comment]
Thanks @sauradip . I will use your pretrained model for cyclegan to generate results

**Comment by chowkamlee81** [comment]
@sauradip , thanks your model worked decently but however results are still manageable. Thanks

**Comment by sauradip** [comment]
Thanks if it has helped you. If you need to make a near perfect night image then you can use a gamma filtering which we have provided in preprocessing file

**Comment by ankur-chr** [comment]
I tried to use the pretrained model. However, the day to night time conversion is producing not so good results. Sometimes the image is too dark (with no added lighting effect: ex car light, street light), or at times the image produced is just a little darker version of the real day time image. Is there a way to fix this?

I used the pytorch colab implementation link from here (https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix) to test this. (link found via the cyclegan link in the readme) 

Also tried using the gamma correction code as suggested. But getting the following error in that:
error: OpenCV(4.1.2) /io/opencv/modules/core/src/matrix.cpp:406: error: (-215:Assertion failed) m.dims >= 2 in function 'Mat'
